resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                    methods:
+                    - GET
+                  property purchase fund release:
+                    href: /lifetime-isa/manager/{lisaManagerReferenceNumber}/accounts/{accountId}/events/fund-releases
+                    methods:
+                    - POST
+                  property purchase extension:
+                    href: /lifetime-isa/manager/{lisaManagerReferenceNumber}/accounts/{accountId}/events/purchase-extensions
+                    methods:
+                    - POST
+                  property purchase outcome:
+                    href: /lifetime-isa/manager/{lisaManagerReferenceNumber}/accounts/{accountId}/events/purchase-outcomes
+                    methods:
+                    - POST
+                  bulk payments:
+                    href: /lifetime-isa/manager/{lisaManagerReferenceNumber}/payments?startDate={startDate}&endDate={endDate}
+                    methods:
+                    - GET
+                  bulk payment breakdown:
+                    href: /lifetime-isa/manager/$lisaManagerReferenceNumber/accounts/{accountId}/transactions/{transactionId}/payments
+                    methods:
+                    - GET
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                invalidLmrn:
+                  $ref: '#/components/examples/invalidLmrn'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                notOpenStatus:
+                  $ref: '#/components/examples/notOpenStatusUnauthorized'
+      deprecated: false
+      security:
+        - user-restricted:
+            - read:lisa
+  /lifetime-isa/manager/{lisaManagerReferenceNumber}/investors:
+    post:
+      tags:
+        - LISA Investors
+      summary: Create a LISA investor
+      description: |
+        Report a new LISA investor to HMRC to generate an investor ID. If the investor already exists, you will get their reference number.
+        
+        ### Test data
+
+        <table style="font-family:Roboto, sans-serif; font-size:14px;">
+            <col width="25%">
+            <col width="35%">
+            <col width="40%">
+            <thead>
+                <tr>
+                    <th>Scenario</th>
+                    <th>Request Payload</th>
+                    <th>Response</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td><p>Request with a valid payload and LISA Manager reference number</p> <p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks